server/src/test/Parsing.test.ts

```diff
@@ -26,6 +26,17 @@ function checkInputValidityForRule(ruleToTest: (parser: YmlParser) => ParserRule
 }
 
 describe('Parsing Tests', () => {
+    describe('intruction_rename rule', () => {
+        it('should parse without errors an `intruction_rename` instruction', (done) => {
+            checkInputValidityForRule((parser) => parser.instruction_rename(), `rename myAttr to myOtherAttr forClass MyClass`);
+            done();
+        });
+        it('should parse without errors an `intruction_rename` instruction using other existing keywords', (done) => {
+            checkInputValidityForRule((parser) => parser.instruction_rename(), `rename rename to forClass forClass Function`);
```

Comment:
`rename rename` ?

Reply:
This is intended `should parse without errors an `intruction_rename` instruction using other existing keywords'. Except if I can't have an attribute named `rename` in a class.

Reply:
Not sure you can. Some words are problematic, I think. Like naming your attribute `null`. That could be tested.

Reply:
This compiles
```
rename    pt to rename forClass MultilingualExpression
rename    rename to forClass forClass MultilingualExpression
rename    forClass to ptLanguage forClass MultilingualExpression
```

Reply:
This does too:
```
rename    pt to null forClass MultilingualExpression
rename    null to forClass forClass MultilingualExpression
rename    forClass to ptLanguage forClass MultilingualExpression
```

Reply:
This compiles too:
```
rename    pt to null forClass MultilingualExpression

MultilingualExpression patate
--> null "Hello"
;
```

Reply:
I'm convinced